testing/features/business_rules/automated_release_process.feature

```diff
@@ -1,27 +0,0 @@
-Feature: Automated Release Process
```

Comment:
I feel we should avoid using this "business rules" directory for design system stuff, but do we want to keep this elsewhere?

Reply:
I just closed the ticket for this work in public website since we'll be reviewing all this once the new docs site is done - so happy for it to go.  We can always come back here if we want it.